src/features/import-assertions.md

```diff
@@ -14,10 +14,10 @@ The new [import assertions](https://github.com/tc39/proposal-import-assertions)
 
 ```javascript
 // foo.json
-{ "answer": 42 }
+{ 'answer': 42 }
```

Comment:
Er, actually this one is JSON so probably should keep double quotes :)

Reply:
Oops, I switched it back here and in the other example, thanks :)